src/ResourceManager/ContainerRegistry/ChangeLog.md

```diff
@@ -0,0 +1,30 @@
+<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
+
+    ## Current Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+
+## Version 1.0.0
```

Comment:
@djyou that's our pattern - any changes going out in the next release will be under the "Current Release" header, and changes that have gone out in prior releases are documented in the respective version header. We run a script during the release process that will move the information from the "Current Release" section into their own version section.